osu.Game/Skinning/SkinnableSprite.cs

```diff
@@ -1,46 +1,102 @@
 // Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
 // See the LICENCE file in the repository root for full licence text.
 
+using System;
+using System.Linq;
 using osu.Framework.Allocation;
+using osu.Framework.Bindables;
 using osu.Framework.Graphics;
 using osu.Framework.Graphics.Sprites;
 using osu.Framework.Graphics.Textures;
+using osu.Game.Configuration;
+using osu.Game.Overlays.Settings;
+using osuTK;
 
 namespace osu.Game.Skinning
 {
     /// <summary>
     /// A skinnable element which uses a stable sprite and can therefore share implementation logic.
```

Comment:
xmldoc probably needs a slight update since this isn't a "stable-only" thing anymore.